packages/@core.keystone/tasks.js

```diff
@@ -121,6 +123,9 @@ function createWorker () {
         console.log('Creating worker process')
     }
 
+    // we needed to prepare keystone to use it inside tasks logic!
+    await prepareKeystoneExpressApp(keystoneModule)
```

Comment:
look's strange, is it correct?

Reply:
Yes. The worker is a separate node process and we need to prepare some keystone internal state like database connection, internal apollo gql setup, and so on ...